### DISCUSSION TITLE
Open Maps for Europe data source

### DISCUSSION THREAD

**Discussion by Jure-BB** [whosonfirst/whosonfirst-sources#194] [General]
Hi,

Is there any plans to import data from Open Maps for Europe dataset?
https://eurogeographics.org/open-maps-for-europe/
https://www.mapsforeurope.org/

I did quick comparision for Slovenia and there are proper admin areas polygons in this dataset, while WOF data contains mostly buffer areas and only few polygons. This data would provide much better PIP for admin areas in Europe.  

EDIT: Screenshots for comparision

**WOF (polygons and buffers):**
![WOF-boundaries-with-buffers](https://user-images.githubusercontent.com/6940357/170109685-38f88321-2847-4648-8925-cf27e418ca14.png)

**WOF (without buffers so polygons are clearly visible):**
![WOF-boundaries](https://user-images.githubusercontent.com/6940357/170109484-fd4d7a54-d450-495f-99c0-969da445e46a.png)

**Eurogeographics:**
![Eurogeographics](https://user-images.githubusercontent.com/6940357/170109625-84aa07c7-895c-4221-a11f-cced1fdad2f8.png)

**Comment by nvkelso** [comment]
Yes, Who's On First would like to pull in newer open datasets from Eurogeographics. We have some older datasets from them incorporated already, but the newer themes and updated themes provide better country coverage and richer attributes than before.

Can you post some screenshots here of your preliminary analysis for Slovenia so its easier for others to follow along and see the value of the proposal, please?

Adding @stepps00 who can help coordinate the work. There are steps that are "easy" and "hard" so some time is involved, but let's look for a way to collaborate on this :)

**Reply by Jure-BB** [reply]
Great. I added screenshots comparing data for Slovenia.

**Comment by nvkelso** [comment]
Looks like a 1000% upgrade in data quality, nice find!

**Reply by Jure-BB** [reply]
Thanks :)

**Comment by stepps00** [comment]
Thanks @Jure-BB 

Who's On First gained new features and geometries for admin0, admin1, and admin2 via the Surveying and Mapping Authority of the Republic of Slovenia's data portal ([issue](https://github.com/whosonfirst-data/whosonfirst-data/issues/755), [source](https://github.com/whosonfirst/whosonfirst-sources/blob/main/sources/svn-sma.json)) back in 2017. In 2020, [this issue](https://github.com/whosonfirst-data/whosonfirst-data/issues/1824) was opened to investigate the newest version of this dataset, but that issue is still open.

I believe Eurogeographics pulls from this same dataset, but we'd have to pull both to compare. I think it's worth reviewing Eurogeographics data, especially at the localadmin and/or locality placetypes. Many locality polygons in Slovenia come from Quattroshapes (see: [Ljubljana](https://spelunker.whosonfirst.org/id/101752073/) as an example), so improving these shapes would benefit Who's On First.

I'll comment in the open `whosonfirst-data` issue with a link to this discussion, along with a list of "to do" items for improving data here..

**Comment by stepps00** [comment]
Also, @Jure-BB - do you happen to have a direct link to the dataset used for these screenshots and a link to the data license?

**Reply by Jure-BB** [reply]
https://www.mapsforeurope.org/access-data

No direct links unfortunately. They email them to you after you make a request and they contain access tokens (so I can't share them).

**Reply by Jure-BB** [reply]
The data on screenshot is from EuroRegionalMap (PolbndA layer)

**Reply by Jure-BB** [reply]
@stepps00 did you manage to get access to the dataset?

License: https://www.mapsforeurope.org/licence

**Comment by nvkelso** [comment]
We lost track of this one, sorry!

Noting here the source includes great data for many EU countries, but this issue is for a single country. Probably we'd want to evaluate on a per country basis, since sometimes WOF already has better data directly from the national government sources.

**Comment by nvkelso** [comment]
In general, WOF prefers to go directly to the national mapping agency. If that's not available then we look for data from an aggregator. We do this for two reasons: It's better to directly source the author of the data and the primary source often has better precision geometries and more interesting properties.

In the case of [Euro](https://www.mapsforeurope.org/access-data) data sometimes that data is downsampled.

I'll keep this discussion open, because there might be good data to pull from in there...

But for Slovenia we do have plans to import from the national mapping agency there:

- https://github.com/whosonfirst-data/whosonfirst-data/issues/1824

**Reply by Jure-BB** [reply]
Thanks for an update! 

I agree that national mapping agency is a better data source than an aggregator like Open Maps for Europe.
